pages/en/deploying/subgraph-studio.mdx

```diff
@@ -64,13 +64,17 @@ If you’d like to test your subgraph before publishing it to the network, you c
 
 ## Publish your Subgraph in Subgraph Studio
 
-You’ve made it this far - congrats! Publishing your subgraph means that an IPFS hash was generated when you deployed the subgraph within the CLI and is stored in the network’s Ethereum smart contracts. In order to publish your subgraph successfully, you’ll need to go through the following steps outlined in this [blog](https://thegraph.com/blog/building-with-subgraph-studio). Check out the video overview below as well:
+You’ve made it this far - congrats!
+
+In order to publish your subgraph successfully, you’ll need to go through the following steps outlined in this [blog](https://thegraph.com/blog/building-with-subgraph-studio).
+
+Check out the video overview below as well:
 
 <VideoEmbed youtube="HfDgC2oNnwo" />
 
 Remember, while you’re going through your publishing flow, you’ll be able to push to either mainnet or Goerli, the testnet we support. If you’re a first-time subgraph developer, we highly suggest you start with publishing to Goerli, which is free to do. This will allow you to see how the subgraph will work in The Graph Explorer and will allow you to test curation elements.
 
-You’ll only be able to index data from mainnet (even if your subgraph was published to a testnet) because only subgraphs that are indexing mainnet data can be published to the network. This is because Indexers need to submit mandatory Proof of Indexing records as of a specific block hash. Because publishing a subgraph is an action taken on-chain, remember that the transaction can take up to a few minutes to go through. Any address you use to publish the contract will be the only one able to publish future versions. Choose wisely!
+You’ll be able to index data from any of the [supported networks](/developing/supported-networks) (even if your subgraph was published to a testnet) because only subgraphs that are indexing mainnet data can be published to the network. This is because Indexers need to submit mandatory Proof of Indexing records as of a specific block hash. Because publishing a subgraph is an action taken on-chain, remember that the transaction can take up to a few minutes to go through. Any address you use to publish the contract will be the only one able to publish future versions. Choose wisely!
```

Comment:
> because only subgraphs that are indexing mainnet data can be published to the network.

This sentence need to be changed or removed 🙏 If removed, we also need to remove "This is because" from the next sentence too.

Reply:
Got it, thank you!